core/sail/fts/elasticsearch/src/main/java/org/eclipse/rdf4j/sail/elasticsearch/ElasticsearchIndex.java

```diff
@@ -173,7 +174,7 @@ public void initialize(Properties parameters)
 		throws Exception
 	{
 		super.initialize(parameters);
-		indexName = parameters.getProperty(INDEX_NAME_KEY, DEFAULT_INDEX_NAME);
+		indexName = parameters.getProperty(INDEX_NAME_KEY, UUID.randomUUID().toString());
```

Comment:
Is it documented that there is a default static index name for elastic search implementations that users can rely on being constant instead of setting it up explicitly in their Sail configuration? 

It isn't clear why users have been focusing on setting different cluster names (which are relatively more difficult to setup and maintain?) compared to adding new indexes to existing clusters.